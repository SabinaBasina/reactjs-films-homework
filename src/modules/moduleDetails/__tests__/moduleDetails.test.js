import reducer from '../moduleDetailsReducers';

describe('reducers', () => {
  it('should return the initial state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot();
  });

  it('should handle IS_READY_TVSHOW', () => {
    expect(
      reducer([], {
        type: 'IS_READY_TVSHOW',
      }),
    ).toMatchSnapshot();
  });

  it('should handle IS_READY_EPISODES', () => {
    expect(
      reducer([], {
        type: 'IS_READY_EPISODES',
      }),
    ).toMatchSnapshot();
  });

  it('should handle SET_TVSHOWEPISODES', () => {
    expect(
      reducer([], {
        type: 'SET_TVSHOWEPISODES',
      }),
    ).toMatchSnapshot();
  });

  it('should handle SET_TVSHOWDETAILS', () => {
    expect(
      reducer([], {
        type: 'SET_TVSHOWDETAILS',
      }),
    ).toMatchSnapshot();
  });
});
