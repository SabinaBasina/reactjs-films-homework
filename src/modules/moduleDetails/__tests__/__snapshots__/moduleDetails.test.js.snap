// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers should handle IS_READY_EPISODES 1`] = `
Object {
  "isReadyEpisodes": undefined,
}
`;

exports[`reducers should handle IS_READY_TVSHOW 1`] = `
Object {
  "isReadyTvShow": undefined,
}
`;

exports[`reducers should handle SET_TVSHOWDETAILS 1`] = `
Object {
  "tvShow": undefined,
}
`;

exports[`reducers should handle SET_TVSHOWEPISODES 1`] = `
Object {
  "tvShowEpisodes": undefined,
}
`;

exports[`reducers should return the initial state 1`] = `
Object {
  "isReadyEpisodes": false,
  "isReadyTvShow": false,
}
`;
